modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm

```diff
@@ -1,8 +1,8 @@
 ////
 //	Typing indicator icon change
 /mob/living/silicon/robot/set_typing_indicator(state, emote)
-	var/static/mutable_appearance/type_indicator = mutable_appearance('modular_skyrat/modules/indicators/icons/typing_indicator.dmi', "borg0", FLY_LAYER)
-	var/static/mutable_appearance/emote_indicator = mutable_appearance('modular_skyrat/modules/indicators/icons/emote_indicator.dmi', "borg0", FLY_LAYER)
+	var/static/mutable_appearance/type_indicator = mutable_appearance('modular_skyrat/modules/indicators/icons/typing_indicator.dmi', "borg0", CURSE_LAYER)
```

Comment:
I'd probably rather they get their own define, just so it's easier to modify in the future. Even if it's just `#define CHAT_INDICATOR_LAYER CURSE_LAYER`